WooCommerce/Classes/Analytics/TracksProvider.swift

```diff
@@ -114,3 +109,26 @@ private extension TracksProvider {
         static let rtlKey               = "is_rtl_language"
     }
 }
+
+extension TracksProvider: WPAnalyticsTracker {
+    public func trackString(_ event: String?) {
+        trackString(event, withProperties: nil)
+    }
+
+    public func trackString(_ event: String?, withProperties properties: [AnyHashable: Any]?) {
+        guard let eventName = event else {
+            DDLogInfo("🔴 Attempted to track an event without name.")
+            return
+        }
+
+        track(eventName, withProperties: properties)
+    }
+
+    public func track(_ stat: WPAnalyticsStat) {
+        track(stat, withProperties: nil)
```

Comment:
nit: maybe add `no op` comment here too? It's not obvious on first glance that method does nothing.

Reply:
Sure!